pkg/machinepool/machinepool_test.go

```diff
@@ -26,6 +28,112 @@ var _ = Describe("Machinepool and nodepool", func() {
 				Value(cmv1.UpgradePolicyStateValueScheduled)).
 				NextRun(date)
 		})
+		It("editAutoscaling should equal nil if nothing is changed", func() {
+			nodepool, err := cmv1.NewNodePool().
+				Autoscaling(cmv1.NewNodePoolAutoscaling().MaxReplica(2).MinReplica(1)).
+				Build()
+			Expect(err).ToNot(HaveOccurred())
+			builder := editAutoscaling(nodepool, 1, 2)
+			Expect(builder).To(BeNil())
+		})
+		It("editAutoscaling should equal the exepcted output", func() {
+			nodepool, err := cmv1.NewNodePool().
+				Autoscaling(cmv1.NewNodePoolAutoscaling().MaxReplica(2).MinReplica(1)).
+				Build()
+			Expect(err).ToNot(HaveOccurred())
+			builder := editAutoscaling(nodepool, 2, 3)
+			asBuilder := cmv1.NewNodePoolAutoscaling().MaxReplica(3).MinReplica(2)
+			Expect(builder).To(Equal(asBuilder))
+		})
+		It("editAutoscaling should equal the exepcted output with no min replica value", func() {
+			nodepool, err := cmv1.NewNodePool().
+				Autoscaling(cmv1.NewNodePoolAutoscaling().MaxReplica(2).MinReplica(1)).
+				Build()
+			Expect(err).ToNot(HaveOccurred())
+			builder := editAutoscaling(nodepool, 0, 3)
+			asBuilder := cmv1.NewNodePoolAutoscaling().MaxReplica(3).MinReplica(1)
+			Expect(builder).To(Equal(asBuilder))
+		})
+
+		It("editAutoscaling should equal the exepcted output with no max replica value", func() {
+			nodepool, err := cmv1.NewNodePool().
+				Autoscaling(cmv1.NewNodePoolAutoscaling().MaxReplica(4).MinReplica(1)).
+				Build()
+			Expect(err).ToNot(HaveOccurred())
+			builder := editAutoscaling(nodepool, 2, 0)
+			asBuilder := cmv1.NewNodePoolAutoscaling().MaxReplica(4).MinReplica(2)
+			Expect(builder).To(Equal(asBuilder))
+		})
+		Context("Prompt For NodePoolNodeRecreate", func() {
+
+			var t *TestingRuntime
+			BeforeEach(func() {
+				t = NewTestRuntime()
+			})
+
+			It("Prompts when the user has deleted a kubelet-config", func() {
+
+				invoked := false
+
+				f := func(r *rosa.Runtime) bool {
+					invoked = true
+					return invoked
+				}
```

Comment:
DRY
This is repeated for each test in this context.
That can be avoided adding this code:
```go
var mockPromptFuncInvoked bool

BeforeEach(func() {
	t = NewTestRuntime()
	mockPromptFuncInvoked = false
})

mockPromptfunc := func(r *rosa.Runtime) bool {
	mockPromptFuncInvoked = true
	return mockPromptFuncInvoked
}
```
Then you can pass `mockPromptFuncInvoked` everywhere `f` is passed and check the value f `mockPromptFuncInvoked` instead of `invoked`